lib/commands/create-piece.js

```diff
@@ -65,7 +65,9 @@ module.exports = function (program, version) {
 
       // Piece widget setup
       // ******************
-      if (options.widgets) {
+      if (options.widgets && majorVersion === '3') {
+        util.log('create-piece', '👉 The --widgets option does not apply to Apostrophe 3 projects.');
```

Comment:
Scaffolding a simple relationship widget for them would be nice, but it could come later.